website/package.json

```diff
@@ -13,10 +13,12 @@
     "write-translations": "docusaurus write-translations",
     "write-heading-ids": "docusaurus write-heading-ids",
     "typecheck": "tsc",
-    "format:check": "prettier --check 'src/**/*.{css,ts,tsx}'",
-    "format:fix": "prettier --write 'src/**/*.{css,ts,tsx}'",
-    "lint:check": "eslint . --ext js,ts,tsx",
-    "lint:fix": "eslint . --fix --ext js,ts,tsx"
+    "format:check": "prettier --check '**/*.{md,js}' 'src/**/*.{css,ts,tsx}'",
+    "format:fix": "prettier --write '**/*.{md,js}' 'src/**/*.{css,ts,tsx}'",
+    "markdownlint:check": "markdownlint-cli2 --config ../.markdownlint.yaml \"**/*.md\" \"#node_modules\"",
+    "markdownlint:fix": "markdownlint-cli2-fix --config ../.markdownlint.yaml \"**/*.md\" \"#node_modules\"",
+    "lint:check": "cd .. && eslint website --ext js,ts,tsx",
+    "lint:fix": "cd .. && eslint website --fix --ext js,ts,tsx"
```

Comment:
it's missing the cache option of parent folder (I don't know for the memory as it was reported by you and I'm not facing the issue)
https://github.com/containers/podman-desktop/blob/main/package.json#L65-L66

Reply:
then again this smaller check target doesn't run out of memory

Reply:
ok for ignoring env for memory but it should use the cache

Reply:
added cache, and a missing markdownlint target (which is failing)